GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/MountTests.cs

```diff
@@ -94,6 +94,56 @@ public void MountSetsCoreHooksPath()
             }
         }
 
+        [TestCase]
+        [Category(Categories.WindowsOnly)] // Only Windows uses GitHooksLoader.exe and merges hooks
+        public void MountMergesLocalPrePostHooksConfig()
+        {
+            // Create some dummy pre/post command hooks
+            string dummyPreCommandHookBin = Path.Combine(this.Enlistment.EnlistmentRoot, "DummyPreCommandHook.bat");
+            this.fileSystem.WriteAllText(dummyPreCommandHookBin, "@echo off && exit 0");

```

Comment:
Looks like the tests don't like this dummy hook:

> Could not execute 'C:\Repos\GVFSFunctionalTests\enlistment\6bba3385e1b54e35a2d4\DummyPostCommandHook.bat'. CreateProcess error (2).

This approach did work on my machine, I will do some digging and see if there's a better way to create a no-op hook in the test.